docs/Architecture/ARCHITECTURE_BUILDING_BLOCKS.md

```diff
@@ -0,0 +1,31 @@
+ANYWAY Data
+================
+
+- CBS
+- Waze
+- OSM
+- News Flash
+- Infographics Data
+- Road Keepers
```

Comment:
Maybe add where these items reside, e.g. database, s3, static files, etc.

Reply:
@ziv17 the attached documents for each logical section aren't enough?

Reply:
Formally yes, but someone who reads it for impression will not stop here and go to each document to find where it resides. A few words of location for each item will give a clearer high-level view.